combat-trainer.lic

```diff
@@ -958,7 +961,10 @@ class SpellProcess
     return if game_state.cast_timer.nil? || (Time.now - game_state.cast_timer) <= 70
 
     game_state.cast_timer = nil
-    fput('release spell') if game_state.casting
+    if game_state.casting
+      fput('release spell')
```

Comment:
I'll go ahead and merge this PR. In a future PR, would you be up for changing this and the `fput('release spell')` in common-arcana to be `bput('release spell')` like in combat-trainer?

Reply:
Yeah, want me to just do that now?